geopy/geocoders/__init__.py

```diff
@@ -169,6 +170,7 @@
     "tomtom": TomTom,
     "what3words": What3Words,
     "yandex": Yandex,
+    "mapquest": MapQuest
```

Comment:
1. The list should be sorted
2. Please take a look at the comment on the 81st line: it's not the only place where a new geocoder should be added.

Reply:
I think I added the geocoder in all the places. Could you confirm?